pkg/syncthing/syncthing.go

```diff
@@ -702,32 +668,36 @@ func (s *Syncthing) Restart(ctx context.Context) error {
 }
 
 // Stop halts the background process and cleans up.
-func (s *Syncthing) Stop(force bool) error {
-	pidPath := filepath.Join(s.Home, syncthingPidFile)
-	pid, err := getPID(pidPath)
-	if os.IsNotExist(err) {
-		return nil
-	}
-
-	if !force {
-		if pid != s.pid {
-			log.Infof("syncthing pid-%d wasn't created by this command, skipping", pid)
-			return nil
-		}
-	}
-
-	if err := s.cleanupDaemon(pid, force); err != nil {
+func (s *Syncthing) Stop(wait bool) error {
```

Comment:
we should add tests for this, especially to validate that we're not killing syncthing's owned by a different up, or other processes running in the machine.